packit_service/worker/jobs.py

```diff
@@ -76,9 +76,16 @@ def get_handlers_for_event(
     """
     Get all handlers that we need to run for the given event.
 
+    We need to return all handler classes that:
+    - can react to the given event
+    - are configured in the package_config (either directly or as a required job)
```

Comment:
Make it explicit that the relationship between these two is 'or':

```suggestion
    We need to return all handler classes that either:
    - can react to the given event OR
    - are configured in the package_config (either directly or as a required job)
```

Reply:
Actually, there should  be `AND`..;-)

I'll make it clear.

Reply:
I inferred the 'or' from 

```python
for job in package_config.jobs:
        if (
            event.db_trigger and event.db_trigger.job_config_trigger_type == job.trigger
        ) or is_trigger_matching_job_config(trigger=event.trigger, job_config=job):
```

What did I miss?

Reply:
But we are going only through the configured jobs.

Those two checks are both about triggers.

Reply:
Duh! Of course. Thanks for clarifying.